scrapers/OldNanny.yml

```diff
@@ -0,0 +1,35 @@
+# This scraper does not parse older scenes that use a different page layout
+name: "OldNanny"
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - oldnanny.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name:
+          selector: //div[@class="col-12 col-xl-4"]//dl[2]//a/text()
```

Comment:
This returns the substudio as it's written: `AgedLove`, `OmaHotel`, `LatinChili` and so on

I'd suggest massaging the data a little more as demonstrated with [this regex](https://regex101.com/r/5UiI39/2)
```
postProcess:
  - replace:
      - regex: ([a-z])([A-Z])
        with: $1 $2
```

Reply:
Clever. Fixed.